lib/fs.js

```diff
@@ -261,6 +265,10 @@ Stats.prototype.toJSON = function toJSON() {
     ino: this.ino,
     size: this.size,
     blocks: this.blocks,
+    atime_msec: this.atime,
+    ctime_msec: this.ctime,
+    mtime_msec: this.mtime,
+    birthtime_msec: this.birthtime,
```

Comment:
Should just be, `atime_msec: this.atime_msec,`, I think? this is in the `toJSON` function, so I figure it should just expose that value.